sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -257,12 +259,22 @@ protected void dispose(AutoCloseable... closeables) {
     protected static Stream<Arguments> messagingEntityWithSessions() {
         return Stream.of(
             Arguments.of(MessagingEntityType.QUEUE, false),
-            Arguments.of(MessagingEntityType.SUBSCRIPTION, false),
+            Arguments.of(MessagingEntityType.SUBSCRIPTION, false)//,
             Arguments.of(MessagingEntityType.QUEUE, true),
             Arguments.of(MessagingEntityType.SUBSCRIPTION, true)
         );
     }
 
+    protected List<ServiceBusMessage> getMessages(String messageId, boolean isSessionEnabled, int numberOfEvents,
```

Comment:
Why create a method that calls into another method rather than using TestUtils as-is?